### PR TITLE
refactoring tests to not use extension methods

### PR DESCRIPTION
**Pull request by jmarolf**
Refactoring our test framework to not use extension methods
@dotnet/roslyn-ide please review

### CODE REVIEW

**Comment by jmarolf** [comment]
Merging preemptively. If there is anything you want me to fix, let me know

**Comment by rchande** [comment]
@jmarolf Is there a way to reduce the amount of typing I have to do for common operations? (`VisualStudio.Editor.SetText`) feels really wordy.

**Comment by rchande** [comment]
@jmarolf, basically, I feel like I'm spending a lot of time writing "VisualStudio.Editor" over and over again.

**Comment by jasonmalinowski** [comment]
@rchande You could make a property `Editor => VisualStudio.Editor` to reduce the typing. We also made a conscious decision that lead to this PR that long-term maintainability of code is more important than saving a second of typing.

(also, our belief is the completion MRU means you're just repeatedly hitting . and that's it. :smile:)

File: src/VisualStudio/IntegrationTest/IntegrationTests/AbstractIntegrationTest.cs [hunk 1]
```diff
@@ -11,24 +11,22 @@ namespace Roslyn.VisualStudio.IntegrationTests
...
     {
-        public readonly VisualStudioInstanceContext VisualStudio;
-        public readonly VisualStudioWorkspace_OutOfProc VisualStudioWorkspaceOutOfProc;
-        public readonly TextViewWindow_OutOfProc TextViewWindow;
+        public readonly VisualStudioInstance VisualStudio;
 
         protected readonly string ProjectName = "TestProj";
         protected readonly string SolutionName = "TestSolution";
 
+        private VisualStudioInstanceContext _visualStudioContext;
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/IntegrationTests/AbstractIntegrationTest.cs]: Should this be readonly too?

**Pull request merged**